flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -277,8 +277,26 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
         }
     }
 
-    private func loadImageList(result: @escaping FlutterResult) {
-      let debugImages = PrivateSentrySDKOnly.getDebugImages() as [DebugMeta]
+    private func loadImageList(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+      var debugImages: [DebugMeta] = []
+      
+      if let arguments = call.arguments as? Set<String> , !arguments.isEmpty{
+          var imagesAddresses: Set<String> = []
+
+          for argument in arguments {
+              if let instructionAddress = UInt64(argument) {
+                let image = SentryDependencyContainer.sharedInstance().binaryImageCache.image(byAddress: instructionAddress)
+                  if let image = image {
+                      let imageAddress = sentry_formatHexAddressUInt64(image.address)!
+                    imagesAddresses.insert(imageAddress)
+                  }
+              }
+          }
+          debugImages = SentryDependencyContainer.sharedInstance().debugImageProvider.getDebugImages(forAddresses: imagesAddresses, isCrash: false) as [DebugMeta]
+      } else {
+          debugImages = PrivateSentrySDKOnly.getDebugImages() as [DebugMeta]
+      }
```

Comment:
this doesn't fall back to `PrivateSentrySDKOnly.getDebugImages()` when either the insruction address parsing throws or the resulting imagesAddresses is still empty

we need to make sure that in all cases this will always return the full debug image list as fallback